### PR TITLE
Adds some miscellaneous equipment to loadout

### PR DESCRIPTION
**Pull request by Recoherent**
## About The Pull Request
Adds black/white gloves, the standard labcoat, trick blindfold, and a laptop to the loadout menu.

## Why It's Good For The Game
atlas and prefab asked for more stuff and i like more stuff

## Changelog

:cl:
add: You can now grab some gloves, labcoats, and even a transparent blindfold in loadouts. Space Radioshack has a new laptop line in stock, too, in case tablets aren't your speed.
/:cl:

### CODE REVIEW

**Comment by Recoherent** [comment]
fun fact, there was some sort of broken loadout subtype grabber for gloves in the accessories file, which i just ASSUME was intended to cause a set of colored gloves to show up? obviously didn't work

File: code/modules/client/loadout/loadout_accessories.dm [hunk 1]
```diff
@@ -67,8 +67,15 @@
 	display_name = "maid neckpiece"
 	path = /obj/item/clothing/neck/maid
 
-/datum/gear/accessory/gloves
-	subtype_path = /datum/gear/accessory/gloves
```
Comment (Ms-Mee) [inline code/modules/client/loadout/loadout_accessories.dm:71]: Glove subtype definition for loading all glove subtypes, including the current evening gloves, is it not?

File: code/modules/client/loadout/loadout_accessories.dm [hunk 2]
```diff
@@ -67,8 +67,15 @@
...
+/datum/gear/accessory/blackgloves
+	display_name = "black gloves"
+	description = "Standard hand coverings for everyday use."
+	path = /obj/item/clothing/gloves/color/black
+
+/datum/gear/accessory/whitegloves
+	display_name = "white gloves"
+	description = "Standard hand coverings for everyday use."
+	path = /obj/item/clothing/gloves/color/white
```
Comment (Ms-Mee) [inline code/modules/client/loadout/loadout_accessories.dm]:
```suggestion
/datum/gear/accessory/gloves/black
	display_name = "black gloves"
	description = "Standard hand coverings for everyday use."
	path = /obj/item/clothing/gloves/color/black

/datum/gear/accessory/gloves/white
	display_name = "white gloves"
	description = "Standard hand coverings for everyday use."
	path = /obj/item/clothing/gloves/color/white
```

File: code/modules/client/loadout/loadout_accessories.dm (hunk 1 above)
Comment (Ms-Mee) [reply code/modules/client/loadout/loadout_accessories.dm:71]: It's there to skip the subtype during loading given it doesn't contain anything and is just there for organization, indeed.

Comment (Ms-Mee) [reply code/modules/client/loadout/loadout_accessories.dm:71]: As in, it skips /datum/gear/accessory/gloves and only loads it's subtypes such as /datum/gear/accessory/gloves/evening

**Pull request merged**